### PR TITLE
fixed image dimensions in chat screen

### LINKED ISSUE #375: Change images dimension in chat screen
**Actual Behaviour**

Currently, Square image is presented in the chat screen

**Expected Behaviour**

display images in different ration eg 2/3 or according to original image ration

**Steps to reproduce it**

Go to chat screen < ask susi for image of batman

**Screenshots of the issue**

simulator screen shot - iphone 8 - 2018-07-06 at 17 30 18

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by imjog**
Fixes #375 

Changes: Changes image dimensions in 3/5 ratio

Screenshots for the change: 
simulator screen shot - iphone 8 - 2018-07-17 at 16 13 54 simulator screen shot - iphone 8 - 2018-07-17 at 16 14 27

### CODE REVIEW

File: Susi/Custom Views/Message Cells/ImageCell.swift [hunk 1]
```diff
@@ -53,7 +53,7 @@ class ImageCell: ChatMessageCell {
 
     func addImageView() {
         textBubbleView.addSubview(imageView)
-        imageView.frame = CGRect(x: 0, y: 0, width: 200, height: 200)
+        imageView.frame = CGRect(x: 0, y: 0, width: 275, height: 165)
```
Comment (chashmeetsingh) [inline Susi/Custom Views/Message Cells/ImageCell.swift]: Still seems like hardcoded values are there. Use ratio like `width * 2/3` or `width * 3/5`.

**Comment by chashmeetsingh** [comment]
Also, regarding the constraints it seems that only a top and left margin is applied ( not applied on the right and bottom) Please check this. @imjog

**Comment by imjog** [comment]
@chashmeetsingh Done! Please check the updated screenshot in the description.

**Comment by chashmeetsingh** [comment]
@imjog please resolve conflicts.

**Pull request merged**